relay/polkadot/src/lib.rs

```diff
@@ -1007,6 +1007,7 @@ pub enum ProxyType {
 	CancelProxy = 6,
 	Auction = 7,
 	NominationPools = 8,
+	ParaRegistration = 9,
```

Comment:
Would it be worth making this also 10 so the encoding is the same between Kusama and Polkadot? (Basti probably disagrees with me on this!)

Reply:
Haha :P Yeah, I don't see any reason for this. They are already different and they are highly chain specific any way.